packages/ozone-helper/ozone-type/ozone/model/ChannelSceneRecurrence.ts

```diff
@@ -0,0 +1,19 @@
+import { Item, OzoneType } from './Item'
+
+@OzoneType('flowr.channel.scene.recurrence')
+export class ChannelSceneRecurrence extends Item {
+	mode?: string
+	weekDays?: Array<Number>
+	months?: Array<Number>
+	days?: Array<Number>
+	color?: String
```

Comment:
Same remark, use IColor type
it's an object of {name: string, value?: string}.

For backward compatibility -> color?: String | IColor